internal/featureflags/temporary.go

```diff
@@ -75,3 +75,10 @@ func UpdateNonUniqueByNameIfSingleOneExists() FeatureFlag {
 		defaultEnabled: true,
 	}
 }
+
+func AccountManagement() FeatureFlag {
```

Comment:
Should we put the loading of accounts under the feature flag as well? 
I guess we're fine with always understanding a 1.1 manifest, as long as we safeguard the actual deployment, so might not be needed.

Reply:
That would've been my idea as well. I think loading does no harm.